src/components/organisms/Dashboard/hooks.ts

```diff
@@ -98,14 +101,23 @@ export default (teamId?: string) => {
     },
     [refetch],
   );
-
-  const { data: projectData } = useGetProjectsQuery({
-    variables: { teamId: teamId ?? "", first: 100 },
+  const initprojectPerPage = 9;
+  const projectPerPage = 6;
```

Comment:
Move these outside of the hooks since they never change.

Is there a reason we have 9 at first and then 6 afterwards? I feel like if 9 was ok on first load, 9 would be fine later since the webpage is already loaded and it'll be easier to fetch than when first loading the page.